src/jit/CMakeLists.txt

```diff
@@ -191,6 +196,10 @@ if (CLR_BUILD_JIT32)
   set(JIT_BASE_NAME ryujit)
 endif()
 
+if(WIN32)
+  add_definitions(-DFX_VER_INTERNALNAME_STR=${JIT_BASE_NAME}.dll)
```

Comment:
Does Details tab show the File Description for these binaries as well or are you choosing to not add them?